docs/en/reference/data-retrieval-and-manipulation.rst

```diff
@@ -379,6 +379,22 @@ Execute the query and fetch all results into an array:
     )
     */
 
+fetchAllKeyValue()
+~~~~~~~~~~~~~~~~~~
+
+Execute the query and fetch the first two columns into an associative array as keys and values respectively:
```

Comment:
> What happens to the rest of the columns?

Nothing. As well as nothing happens to other columns when the first column is fetched via `fetchFirstColumn()`.

> it should be put into the documentation

How would you put it?

Reply:
```
.. note::
   Any additional column to the key and value columns will be ignored by the method.
```

Reply:
While I agree that this note is better than nothing, I wouldn't like the users to rely on this behavior just because it's currently implemented. How about this:
> All additional columns will be ignored and are only allowed to be selected by DBAL for its internal purposes.